auth0/resource_auth0_client.go

```diff
@@ -13,6 +13,14 @@ func newClient() *schema.Resource {
 		Read:   readClient,
 		Update: updateClient,
 		Delete: deleteClient,
+		Importer: &schema.ResourceImporter{
+			State: func(data *schema.ResourceData, i interface{}) ([]*schema.ResourceData, error) {
+				results := make([]*schema.ResourceData, 1, 1)
+				results[0] = data
+				readClient(data, i)
+				return results, nil
+			},
+		},
```

Comment:
Can we define this as a function as follows?
```
func  importClient(d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error)
```
 It would be more consistent with `{create,read,update,delete}Resource`.